frontend/src/components/Navbar/NavbarWorksheetSearch.tsx

```diff
@@ -189,6 +193,24 @@ export function NavbarWorksheetSearch() {
 
   const [currentFriendNetID, setCurrentFriendNetID] = useState('');
 
+  const [selectedWorksheetName, setSelectedWorksheetName] = useState('');
+
+  const [newWorksheetName, setNewWorksheetName] = useState('');
+
+  const [addWorksheetPlaceHolder, setAddWorksheetPlaceHolder] = useState(
+    worksheet_options.length >= 20
+      ? 'Reached maximum number of worksheets'
+      : 'Add worksheet',
+  );
+
+  const [boolWorksheetsNumber, setBoolWorksheetsNumber] = useState(
```

Comment:
Don't call a variable "bool". Call it something like `maxWorksheetsNumExceeded` or something like that